### DISCUSSION TITLE
Is there a test report about the latency of getting a small block (64K/128K) from a big file (256M)?

### DISCUSSION THREAD

**Discussion by yiluoya** [juicedata/juicefs#2540] [General]
有没有 获取一个page （64k/128k）这种接口的时延测试报告?  目前测试的oss 和 obs的性能，想了解下juiceFS 有没有这方面的性能测试报告。

**Comment by SandyXSD** [comment]
Not sure what you're really interested in, but you can find benchmark results here: https://juicefs.com/docs/zh/community/benchmark/

**Comment by yiluoya** [comment]
> Not sure what you're really interested in, but you can find benchmark results here: https://juicefs.com/docs/zh/community/benchmark/

thank you very much for your reply， in my case，only get a part from a big file using startpos and offset。  Only case about how long will spend getting a small page/block ( 64k/128 )  from a big file(256M)。

**Reply by davies** [reply]
you can test random read (64K) using `fio`

**Comment by SandyXSD** [comment]
There is no test result for this case, you'd better test it by yourself.
Generally, JuiceFS applies an adaptive read-ahead algorithm to prefetch objects to the local disk cache. So if your application does sequential read, the latency should be quite small. If it does random small reads, the latency depends on that of the object storage.
